### ISSUE TITLE
tf.dll with CPU

### ISSUE THREAD

**Issue by zengzhiw** [BowieHsu/Tensorflow-windows-dll#5]
有没有编译好的CPU版的dll？非常感谢！

**Comment by kalluwa** [comment]
我把编译好的版本放在这里，供有需要的朋友使用

I have compiled the tensorflow1.2 with only cpu, feel free to use

https://pan.baidu.com/s/1j3M076YEwlInZcez47fdfQ

**Comment by nikita-astronaut** [comment]
@kalluwa , is this C or C++ lib? If C, can you please write the line how to compile code that uses this library on Windows? I have MSVC2015

**Comment by kalluwa** [comment]
@nikita-astronaut  yes, it's c++ lib. you can follow the readme of this project and replace the three files "tensorflow.lib","tensorflow.dll","libprotobuf.lib" . and that's all.
<vs2015 **update3**>.

**Comment by anguoyang** [comment]
@kalluwa 你编译的这个版本好像不支持CPU AVX/SSE2/SSE4？期待编译过程

**Comment by kalluwa** [comment]
@anguoyang  https://www.zybuluo.com/kalluwa/note/1110511

**Comment by anguoyang** [comment]
@kalluwa 谢谢，我也是用cmake编译的，加上了avx等选项，编译链接本身没问题，但是生成的tensorflow.lib被console程序链接的时候，总是说tensorshape undefined which referenced in xxx，也就是说链接的时候找不到这个，很奇怪别的类可以，就这个tensorshape不行，郁闷：）

**Comment by anguoyang** [comment]
我用的是官方正式release的1.8版本

**Comment by Neargye** [comment]
@zengzhiw Hi, my build https://github.com/Neargye/hello_tf_win_c_api, maybe this will help.

**Comment by CasonTsai** [comment]
@kalluwa 您好，请问该如何调用您编译好的库，是只要下载tensorflow的源码+您编译好的库就可以了，，还需要编译其他的库么

**Comment by kalluwa** [comment]
@caishengzao   你按照这个repo的readme 走下来，最后替换掉BowieHsu 的dll就可以了

**Comment by CasonTsai** [comment]
@kalluwa 
![image](https://user-images.githubusercontent.com/18262642/50334103-3cf23f00-0542-11e9-8078-3cea323f0d9e.png)主要是一直打不开eigen,可是头文件包含的路径是正确的，，，
